### DISCUSSION TITLE
Holoparasite balancing

### DISCUSSION THREAD

**Discussion by B0xy1** [ParadiseSS13/Paradise#18945] [Documents]
So this has been on my mind for a long time as somebody who uses holoparasites often. 

Ranged is the only viable one compared to the other ones  a bad player can preform decent playing it  and a good player can wipe out a security team. 

All the other holoparas preform poorly in comparison making this the one i always choose and most players in the know seem to choose.
I feel like big part of the problem is that most players have never played holoparasites and its not intuitively easy  to understand how to play explosive holopara or electric holopara and there is little help and instruction in the wiki.
Making message in bald big  red to every holoparasite with tips on how to play would go a long way on solving this issue.
Other then that nerfing Ranged holopara or  Boosting the other holoparas my personal favorite feels needed for balancing.  

Try to set up a system calculating all the times people get holopara and which ones they get  i tell you ranged is the most viable option.

**Comment by hal9000PR** [comment]
I agree that some holoparas are much stronger than others, and i would appreciate any PR that brings the ranged holo down a peg and the others up one or two
